src/DynamoCoreWpf/Interfaces/IWatchHandler.cs

```diff
@@ -170,8 +172,18 @@ private static string ToString(object obj)
         public WatchViewModel Process(dynamic value, ProtoCore.RuntimeCore runtimeCore, string tag, bool showRawData, WatchHandlerCallback callback)
         {
             return Object.ReferenceEquals(value, null)
-                ? new WatchViewModel(NULL_STRING, tag)
+                ? new WatchViewModel(NULL_STRING, tag, OnRequestSelectGeometry)
                 : ProcessThing(value, runtimeCore, tag, showRawData, callback);
         }
+
+        public event Action<string> RequestSelectGeometry;
+
+        private void OnRequestSelectGeometry(string path)
```

Comment:
`RequestSelectGeometry` is event and because of that it cannot be passed as a method/constructor parameter